### PR TITLE
ogre3d: update to 13.4.3

### PR DESCRIPTION
**Pull request by MehdiChinoune**

### CODE REVIEW

**Comment by mmuetzel** [comment]
Maybe, it would be possible to avoid the out-of-memory issue on MINGW32 by skipping to build the Python module for that environment. Similar to #11849.

**Comment by MehdiChinoune** [comment]
> Maybe, it would be possible to avoid the out-of-memory issue on MINGW32 by skipping to build the Python module for that environment. Similar to #11849.

It succeeds on my side.
The ultimate solution is to drop 32biy entirely. Unfortunately instead of dropping MINGW32 we got CLANG32. What a joke!

File: mingw-w64-ogre3d/014-install-pkhconfig-and-fix-plugin-extension.patch [hunk 1]
```diff
@@ -0,0 +1,17 @@
+--- a/CMake/ConfigureBuild.cmake
```
Comment (mmuetzel) [inline mingw-w64-ogre3d/014-install-pkhconfig-and-fix-plugin-extension.patch:1]: Nit: Typo in file name. `pkhconfig` -> `pkgconfig`

**Comment by MehdiChinoune** [comment]
As always, Not a single word if it is accepted, rejected or need some modifications.

**Pull request closed without merging**